src/content/guides/how-to-create-an-ethereum-account/index.md

```diff
@@ -51,17 +51,17 @@ Once you have saved your recovery phrase you should see your wallet dashboard wi
 
 No, just like online banking you can have many different accounts all stored in one wallet application. Your 12 or 24 word phrase secures them all: it is like the seed of a big tree (which is why you must keep it safe). Each branch of the tree holds a key, and every key is one of your accounts. If you lose access to the wallet (i.e. your tree gets cut down), you can always restore all your different accounts using other software and the same seed (which will always grow the same tree).
 
-### Can I send bitcoin to an Ethereum address, or ether to a Bitcoin address?
+### Can I send Bitcoin to an Ethereum address, or Ether to a Bitcoin address?
```

Comment:
```suggestion
### Can I send bitcoin to an Ethereum address, or ether to a Bitcoin address?
```

Uppercase 'Bitcoin' refers to the network, bitcoin (the currency) uses a lowercase b, unless at the beginning of a sentence. Same rule for ether.

Reply:
Okay, got it. Thanks for the information.